common/changes/@itwin/core-bentley/bp-clear-vite-cache_2025-02-27-18-46.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-bentley",
+      "comment": "Clean cache on every test run to make sure linked dependencies are up to date",
```

these change files are supposed to be user facing, there were no changes in our api that we need to call out here

```suggestion
      "comment": "",
```